content/blog/2018-08-14-where-to-bird.md

```diff
@@ -0,0 +1,365 @@
+---
+title: "Where to go observe birds in Radolfzell? An answer with R and open data"
+slug: where-to-bird
+authors:
+  - name: Maëlle Salmon
+    url: https://masalmon.eu/
+  - name: Mark Padgham
+    url: https://github.com/mpadge
+date: 2018-08-14
+preface: The blog post series corresponds to the material for a talk Maëlle wll give at the [Animal Movement Analysis summer school in Radolfzell, Germany](http://animove.org/courses/animove-2018/) on [September the 12th](http://animove.org/animove-2019-evening-keynotes/), in a Max Plant Institute of Ornithology.
+tags:
+- osmdata
+- osmplotr
+- opencage
+- bbox
+- magick
+- birder
+output: 
+  md_document:
+    variant: markdown_github
+    preserve_yaml: true
+---
+
+This post is the 1st post of a series showcasing various rOpenSci
+packages as if Maëlle were a birder trying to make the most of R in
+general and rOpenSci in particular. Although the series use cases will
+mostly feature *birds*, it’ll be the occasion to highlight rOpenSci’s
+packages that are more widely applicable, so read on no matter what your
+field is! Moreoever, each post should stand on its own.
+
+A logical first step in this birder’s guide to rOpenSci is to use R to
+find where to observe birds! In this blog post, we shall use **rOpenSci
+packages accessing open geographical data** to locate and visualize
+where to observe birds near a given location.
+
+### First of all, where are we now?
+
+The Max Planck Institute for Ornithology (henceforth shortened to MPI),
+where Maëlle will give a talk is located at *Am Obstberg 1 78315
+Radolfzell*. Let’s geolocate it using [rOpenSci’s `opencage`
+package](https://github.com/ropensci/opencage) that interfaces the
+[OpenCage Geocoder](https://opencagedata.com/), [commercial service
+based on open data](https://opencagedata.com/credits). Choosing to get
+only one result, we’ll get the best one.
```

Comment:
"Choosing to get only one result, we’ll get the best one."
not sure what that means

Reply:
Am rephrasing it to "When choosing to get only one result via `limit = 1`, one gets what the API considers to be the best one."